Assets/MixedRealityToolkit/_Core/Utilities/Lines/DataProviders/BaseMixedRealityLineDataProvider.cs

```diff
@@ -431,9 +431,9 @@ public Vector3 GetUnClampedPoint(float normalizedLength)
         public Vector3 GetPoint(int pointIndex)
         {
             if (pointIndex < 0 || pointIndex >= PointCount)
-            {
-                Debug.LogError("Invalid point index");
-                return Vector3.zero;
+			{
+				Debug.LogError("Invalid point index: " + pointIndex, this);
```

Comment:
Formatting seems a bit broken here. Did you change to tabs instead of spaces?

Reply:
Yeah, I'll fix that, thought I got all those spots.

Is there any way to configure VS to use tabs vs spaces depending on the project?